libs/schemas/src/queries/community.schemas.ts

```diff
@@ -56,8 +56,8 @@ export const GetStakeTransaction = {
       stake_direction: z.string(),
       community: z.object({
         id: z.string(),
-        default_symbol: z.string(),
-        icon_url: z.string(),
+        default_symbol: z.string().nullable(),
+        icon_url: z.string().nullable(),
```

Comment:
We should use `nullish` so that the properties can be entirely absent and still be valid.

Reply:
cc @kurtisassad to resolve